server/status.go

```diff
@@ -148,15 +152,41 @@ func (s *statusServer) handleLocalStacks(w http.ResponseWriter, r *http.Request)
 // handleNodeStatus handles GET requests for node status.
 func (s *statusServer) handleNodeStatus(w http.ResponseWriter, r *http.Request) {
 	// TODO(shawn) parse node-id in path
-	nodes := &status.NodeList{}
-	b, contentType, err := util.MarshalResponse(r, nodes, []util.EncodingType{util.JSONEncoding})
+
+	startKey := engine.KeyStatusNodePrefix
```

Comment:
I think you actually want to make this a separate handler. `handleNodeStatus` refers to a single node. `handleNodesStatus` would give you the list. Note the difference between `/_status/local/` and `/_status/local/stacks`.